### DISCUSSION TITLE
Hvordan deler vi inspirasjon til tekst og mikrokopi?

### DISCUSSION THREAD

**Discussion by onordbo** [fremtind/jokul#2884] [Annet]
UX Writing kom opp som tema i dagens forum. Hvordan kan vi samle, gruppere og dele gode eksempler som alle kan lære av? Kjør debatt!

**Comment by onordbo** [comment]
Det har også blitt foreslått at vi burde kjøre en workshop rundt temaet. Svar/kommenter på denne kommentaren om du er interessert i å bli med på det!

**Reply by kristineSorensen** [reply]
First

**Reply by shvoldum** [reply]
Second

**Reply by ivnai** [reply]
interessert

**Reply by karimarkhus** [reply]
Blir med 👍
